azext_hanaonazure/_params.py

```diff
@@ -11,4 +11,11 @@
 def load_arguments(self, _):
     with self.argument_context('hanainstance') as c:
         c.argument('resource_group_name', arg_type=resource_group_name_type)
-        c.argument('instance_name', options_list=['--instance-name', '-n'], help="The name of the SAP HANA instance", id_part='name')
\ No newline at end of file
+        c.argument('instance_name', options_list=['--instance-name', '-n'], help="The name of the SAP HANA instance", id_part='name')
+        c.argument('hana_vnet', options_list=['--hana-vnet', '-hv'], help="ARM ID of an Azure Vnet with access to the HANA instance.")
+        c.argument('hana_hostname', options_list=['--hana-hostname'], help="Hostname of the HANA Instance blade.")
+        c.argument('hana_instance_num', options_list=['--hana-instance-num'], help="A number between 00 and 99, stored as a string to maintain leading zero.")
+        c.argument('db_container', options_list=['--db-container'], help="Either single or multiple depending on the use of MDC(Multiple Database Containers). Possible values include: 'single','multiple'")
```

Comment:
I suppose this refers to the (boolean) parameter `HanaMdc`; however, `db_container`, is misleading as it suggests that a specific DB container is expected (rather than a boolean whether MDC is used or not).

Reply:
I have renamed it